src/StructuredLogger.Tests/ConditionParserTests.cs

```diff
@@ -7,6 +8,12 @@ namespace StructuredLogger.Tests
 {
     public class ConditionParserTests
     {
+        public ConditionParserTests()
+        {
+            CultureInfo.DefaultThreadCurrentCulture = new("en-US");

```

Comment:
Could you explain what are these for?

Reply:
Some tests fail When unit tests run on a computer with a different culture than English - which involves different decimal separator.